packages/common/src/controllers/submissionController.ts

```diff
@@ -16,55 +16,57 @@ const controller = (dependencies: Dependencies) => {
 			try {
 				const categoryId = Number(req.params.categoryId);
 				const files = req.files as Express.Multer.File[];
+				const organization = req.body.organization;
 
 				logger.info(
 					LOG_MODULE,
-					`Upload Submission Request categoryId '${categoryId}' files '${files?.map((f) => f.originalname)}'`,
+					`Upload Submission Request categoryId '${categoryId}' organization '${organization}' files '${files?.map((f) => f.originalname)}'`,
 				);
 
 				if (isNaN(categoryId)) {
 					throw new BadRequest('Invalid categoryId number format');
 				}
 
+				if (!organization) {
+					throw new BadRequest('Request is missing `organization` parameter.');
+				}
+
 				if (!files || files.length == 0) {
 					throw new BadRequest('Request is missing attach `files`');
 				}
 
 				const fileErrors: BatchError[] = [];
-				const rawSubmissionEntities: SubmissionEntity[] = [];
+				const validFiles: Express.Multer.File[] = [];
 
 				for (const file of files) {
 					try {
 						validateTsvExtension(file);
-
-						const parsedData = await tsvToJson(file.path);
-						rawSubmissionEntities.push({
-							batchName: file.originalname,
-							creator: '', //TODO: get user from auth
-							records: parsedData,
-						});
+						validFiles.push(file);
 					} catch (error) {
 						logger.error(LOG_MODULE, `Error processing file '${file.originalname}'`, getErrorMessage(error));
 
 						const batchError: BatchError = {
+							type: BATCH_ERROR_TYPE.INVALID_FILE_EXTENSION,
 							message: getErrorMessage(error),
-							type: BATCH_ERROR_TYPE.TSV_PARSING_FAILED,
 							batchName: file.originalname,
 						};
 
 						fileErrors.push(batchError);
 					}
 				}
 
-				const resultSubmission = await service.uploadSubmission(rawSubmissionEntities, categoryId);
+				const resultSubmission = await service.uploadSubmission(validFiles, categoryId, organization);
 
-				let status = 422;
-				if (resultSubmission.successful && fileErrors.length == 0 && resultSubmission.batchErrors.length == 0) {
-					status = 201;
+				if (fileErrors.length == 0 && resultSubmission.batchErrors.length == 0) {
 					logger.info(LOG_MODULE, `Submission uploaded successfully`);
 				} else {
 					logger.error(LOG_MODULE, 'Found some errors processing this request');
 				}
+
+				// HTTP 200 status Submission Accepted
+				const status = 200;
```

Comment:
this `status` variable is not needed anymore here. I left it here because on previous implementation I was changing it conditionally to return different status. This is now removed to just `res.status(200)`